.github/workflows/ci.yml

```
@@ -288,13 +355,8 @@ jobs:
           username: ${{ secrets.GH_USERNAME }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Pull, launch, and validate MAPDL service
-        run: .ci/start_mapdl.sh
-        env:
-          LICENSE_SERVER: ${{ secrets.LICENSE_SERVER }}
-          MAPDL_IMAGE: ${{ env.DOCKER_PACKAGE }}:${{ matrix.mapdl-version }}
-
       - name: Unit Testing
+        timeout-minutes: 9
```

9 just seems a little aggressive.

```suggestion
        timeout-minutes: 15
```